tutordiscovery/templates/discovery/build/discovery/Dockerfile

```diff
@@ -60,9 +60,18 @@ RUN {% if is_buildkit_enabled() %}--mount=type=cache,target=/openedx/.cache/pip,
     # Use redis as a django cache https://pypi.org/project/django-redis/
     django-redis==5.2.0 \
     # uwsgi server https://pypi.org/project/uWSGI/
-    uwsgi==2.0.21
+    uwsgi==2.0.21 \
+    # Open edX Atlas tranlsation management tool https://pypi.org/project/openedx-atlas/
+    openedx-atlas'<1.0.0'
+
+# Update i18n and collect static assets
+{% if DISCOVERY_ATLAS_PULL %}
```

Comment:
Pulls all languages. 

Your point that atlas should pull only relatively complete languages is probably a feature in `atlas` which I've added as a suggestion:

 - https://github.com/openedx/openedx-atlas/issues/44